lib/generators/flipper/templates/sequel_migration.rb

```diff
@@ -0,0 +1,24 @@
+class CreateFlipperTablesSequel < Sequel::Migration
```

Comment:
We can do that in subsequent PRs if you would prefer. I don't see it as a blocker to merging, just curious in general.